examples/semi_supervised_vae/vae_ssl_rws.py

```diff
@@ -130,11 +137,18 @@ def log_joint(observed):
     proposal = unlabeled_proposal(x_unlabeled_ph, n_y, n_z, n_particles)
     qy_samples, log_qy = proposal.query('y', outputs=True, local_log_prob=True)
     qz_samples, log_qz = proposal.query('z', outputs=True, local_log_prob=True)
-    unlabeled_cost, unlabeled_log_likelihood = zs.rws(
-        log_joint, {'x': x_unlabeled_obs},
-        {'y': [qy_samples, log_qy], 'z': [qz_samples, log_qz]}, axis=0)
-    unlabeled_cost = tf.reduce_mean(unlabeled_cost)
-    unlabeled_log_likelihood = tf.reduce_mean(unlabeled_log_likelihood)
+
+    unlabeled_klpq_obj = zs.variational.klpq(
+        log_joint, observed={'x': x_unlabeled_obs},
+        latent={'y': [qy_samples, log_qy],
+                'z': [qz_samples, log_qz]}, axis=0)
+    unlabeled_variational_cost = tf.reduce_mean(unlabeled_klpq_obj.rws())
```

Comment:
klpq_cost is better

Reply:
Have you tested the program to check whether the results in first epoches doesn't change too much?

Reply:
Yes, 

Old Version, 
>Epoch 1 (24.7s), Lower bound: labeled = -169.6367950439453, unlabeled = -176.50120544433594 Accuracy: 98.96%
>Epoch 2 (24.2s), Lower bound: labeled = -107.6112060546875, unlabeled = -139.37413024902344 Accuracy: 100.00%

Current Version,
>Epoch 1 (25.0s), Lower bound: labeled = -169.51016235351562, unlabeled = -176.43612670898438 Accuracy: 99.18%
>Epoch 2 (24.6s), Lower bound: labeled = -107.60102844238281, unlabeled = -139.04464721679688 Accuracy: 100.00%